.github/workflows/tests-ui.yml

```diff
@@ -0,0 +1,31 @@
+name: UI & a11y tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  tests-ui:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Start app
+        run: |
+          cp .env.sample .env
+          docker compose up --detach app
+
+      - name: Run Lighthouse tests for a11y
+        uses: treosh/lighthouse-ci-action@12.1.0
+        with:
+          urls: http://localhost:8000
```

Comment:
Glad you saw the issue with `/help` :grin: 

The other issue is, it isn't certain that `8000` will be the port on `localhost`.

We may need to introduce an environment variable like `DJANGO_LOCAL_PORT`, and use that in the compose service (with a default value) to ensure we have a known port binding.

Related, I added `DJANGO_DEBUG=true` as default in the environment file, but maybe this shouldn't be in there at all. We want the lighthouse tests to run on the non-debug / real app.

Reply:
Yep, I had overlooked the `/help` url 😅 

Thanks, I modified `ports` in the compose file (following the way we do it in Benefits) so that we use `DJANGO_LOCAL_PORT` on `localhost` if it's available, and if it's not, it defaults to `8000`.

Right, I removed `DJANGO_DEBUG=true` from the environment file. I think this shouldn't be a problems since (at least until now) we only use `DJANGO_DEBUG` when starting a debugger (and we already set it in the [debugger configuration](https://github.com/compilerla/pems/blob/main/.vscode/launch.json)) and in `settings.py` (and it gets a default value of `False` so that's ok). If the user wants to set it to `True`, they can override it in their `.env` file.

Reply:
Hmm, `tests-ui` is still failing even after modifying `compose.yml` and removing `DJANGO_DEBUG=true` from `.env.sample`. I'm looking at it some more to see if there's something else I may have missed.

Reply:
Ah that's what it is!
I'm wondering, should I use the admin route just to see if the Lighthouse tests workflow runs, and then we can update `urls` with actual meaningful urls?